nym-vpn-x/src-tauri/src/http.rs

```diff
@@ -4,7 +4,7 @@ use once_cell::sync::Lazy;
 use reqwest::Client;
 use tracing::error;
 
-pub static HTTP_CLIENT: Lazy<Client> = Lazy::new(|| {
+pub static _HTTP_CLIENT: Lazy<Client> = Lazy::new(|| {
```

Comment:
Does it need to public? Looks like you're trying to hide it by prepending it with `_`.

Reply:
I prepended it with `_` because it's not used anymore, but as it could be still useful for the future, I want to keep the declaration

Reply:
Why not deleting the dead code? We can always revive the old code from git history of needed.